roles/octavia/tasks/generate_resources.yml

```diff
@@ -152,3 +152,89 @@
     cloud: atmosphere
     image: "{{ octavia_amphora_image_name }}"
   register: _octavia_amphora_image
+
+- name: Deploy Amphora jumpbox
+  run_once: true
+  block:
+    - name: Create amphora ssh key
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          - apiVersion: secretgen.k14s.io/v1alpha1
+            kind: SSHKey
+            metadata:
+              name: amphora-ssh-key
+              namespace: openstack
+            spec: {}
+
+    - name: Wait until ssh key is reconciled
+      kubernetes.core.k8s_info:
+        api_version: apps/v1
+        kind: SSHKey
+        name: amphora-ssh-key
+        namespace: openstack
+        wait_sleep: 1
+        wait_timeout: 60
+        wait: true
+        wait_condition:
+          type: ReconcileSucceeded
+          status: true
```

Comment:
We don't need a separate task to wait over here, see this:

https://github.com/vexxhost/atmosphere/blob/main/roles/rabbitmq/tasks/main.yml#L49-L53